### PR TITLE
Updated MidnightControls version

### PR DESCRIPTION
**Pull request by zSatur**

### CODE REVIEW

File: src/wiki/getting-started/controller-support.md [hunk 1]
```diff
@@ -22,9 +22,9 @@ Finally, if you don't have one already, you must create an **instance** with the
 
 Before you continue, ensure that the correct version of the **Fabric API** mod on **Fabric** or the **Quilted Fabric API** mod on **Quilt** for your instance is installed.
 
-### <img src="https://cdn-raw.modrinth.com/data/bXX9h73M/icon.svg" height="20">  MidnightControls (for Minecraft Versions 1.18 to 1.19.2)
+### <img src="https://cdn-raw.modrinth.com/data/bXX9h73M/icon.svg" height="20">  MidnightControls (for Minecraft Versions 1.18 to 1.19.4)
```
Comment (Scrumplex) [inline src/wiki/getting-started/controller-support.md:25]:
```suggestion
### <img src="https://cdn-raw.modrinth.com/data/bXX9h73M/icon.svg" height="20">  MidnightControls (for Minecraft versions 1.18 or newer)
```

File: src/wiki/getting-started/controller-support.md [hunk 2]
```diff
@@ -22,9 +22,9 @@ Finally, if you don't have one already, you must create an **instance** with the
...
 
-For Minecraft 1.18 to 1.19.2, we recommend [**MidnightControls**](https://modrinth.com/mod/midnightcontrols), an updated fork of LambdaControls.
+For Minecraft 1.18 to 1.19.4, we recommend [**MidnightControls**](https://modrinth.com/mod/midnightcontrols), an updated fork of LambdaControls.
```
Comment (Scrumplex) [inline src/wiki/getting-started/controller-support.md:27]:
```suggestion
For Minecraft 1.18 or newer, we recommend [**MidnightControls**](https://modrinth.com/mod/midnightcontrols), an updated fork of LambdaControls.
```

**Review by Scrumplex — COMMENTED** [review]
Hmm. Maybe we can just get rid of the upper bound at this point.

**Comment by zSatur** [comment]
Updated Lambdacontrols version and did what you did. I have no clue how github works but yes

File: src/wiki/getting-started/controller-support.md [hunk 3]
```diff
@@ -38,7 +38,7 @@ It can be installed using Prism Launcher's mod downloader function through eithe
 
 ## <img src="https://avatars0.githubusercontent.com/u/1390178?s=400&v=4" height="20"> Forge
 
-### <img src="https://raw.githubusercontent.com/MrCrayfish/Controllable/6caef1a4ac113e5c6ac1d1abde0f0cabc3e6ad97/src/main/resources/controllable_icon.png" height="20">  Controllable (for Minecraft Versions 1.12.2 to 1.19.2)
+### <img src="https://raw.githubusercontent.com/MrCrayfish/Controllable/6caef1a4ac113e5c6ac1d1abde0f0cabc3e6ad97/src/main/resources/controllable_icon.png" height="20">  Controllable (for Minecraft Versions 1.12.2 to newer)
```
Comment (Scrumplex) [inline src/wiki/getting-started/controller-support.md:41]:
```suggestion
### <img src="https://raw.githubusercontent.com/MrCrayfish/Controllable/6caef1a4ac113e5c6ac1d1abde0f0cabc3e6ad97/src/main/resources/controllable_icon.png" height="20">  Controllable (for Minecraft Versions 1.12.2 or newer)
```

File: src/wiki/getting-started/controller-support.md [hunk 4]
```diff
@@ -22,13 +22,13 @@ Finally, if you don't have one already, you must create an **instance** with the
 
 Before you continue, ensure that the correct version of the **Fabric API** mod on **Fabric** or the **Quilted Fabric API** mod on **Quilt** for your instance is installed.
 
-### <img src="https://cdn-raw.modrinth.com/data/bXX9h73M/icon.svg" height="20">  MidnightControls (for Minecraft Versions 1.18 to 1.19.2)
+### <img src="https://cdn-raw.modrinth.com/data/bXX9h73M/icon.svg" height="20">  MidnightControls (for Minecraft Versions 1.18 to newer)
```
Comment (Scrumplex) [inline src/wiki/getting-started/controller-support.md:25]:
```suggestion
### <img src="https://cdn-raw.modrinth.com/data/bXX9h73M/icon.svg" height="20">  MidnightControls (for Minecraft Versions 1.18 or newer)
```

**Review by Scrumplex — COMMENTED** [review]
small typo

**Review by Scrumplex — APPROVED** [review]
Thanks!

**Pull request merged**